core/common/src/LocalDate.kt

```diff
@@ -218,7 +218,7 @@ public expect class LocalDate : Comparable<LocalDate> {
 
     /**
      * Compares `this` date with the [other] date.
-     * Returns zero if this date represents the same day as the other (i.e., equal to other),
+     * Returns zero if this date represents the same day as the other (for example, equal to other),
```

Comment:
That's not "e.g.", that's "i.e."
We could replace this with something like "in other words" if "i.e." is undesirable.

Reply:
Thank you for the clarification, here are several options:
(meaning they're equal to each other)
(with equal values)
(equal to one another)